pkg/operation/botanist/monitoring.go

```diff
@@ -541,3 +582,14 @@ func (b *Botanist) DeleteSeedMonitoring(ctx context.Context) error {
 
 	return kutil.DeleteObjects(ctx, b.K8sSeedClient.Client(), objects...)
 }
+
+// ObservedComponents is a struct from all observed pods
+type ObservedComponents struct {
+	ObservedPods []ObservedPods `yaml:"observedPods"`
+}
+
+// ObservedPods holds the Pods which are monitored through dashboards
+type ObservedPods struct {
```

Comment:
Why plural when the struct represents since ObservedPod?

```suggestion
type ObservedPod struct {
```